src/SDKs/CognitiveServices/dataPlane/Vision/CustomVision/Training/Microsoft.Azure.CognitiveServices.Vision.CustomVision.Training.csproj

```diff
@@ -6,15 +6,15 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Vision.CustomVision.Training</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services CustomVision Training APIs.</Description>
-    <VersionPrefix>1.0.2-preview</VersionPrefix>
+    <Version>0.10.0-preview</Version>
```

Comment:
Don't think you can go back versions, also differs from the private repo entry, please check the versioning here